### ISSUE TITLE
Error running newspaper3k

### ISSUE THREAD

**Issue by mansoorfayyaz** [codelucas/newspaper#360]
I succesfully installed the newspaper3k ver 0.1.9 package in pyCharm community edition. But while running the given example, i ran into following error. Plz advise how to resolve this.

from newspaper import Article
url = 'http://fox13now.com/2013/12/30/new-year-new-laws-obamacare-pot-guns-and-drones/'
article = Article(url)

returned the following error:

File "E:\Python Workspace\TestProject\TestProjectVirtualEnv\lib\site-packages\PIL\Image.py", line 56, in <module>
    from . import _imaging as core
ImportError: DLL load failed: The specified procedure could not be found.

Process finished with exit code 1

Thanks. Your help is highly appreciated.

**Comment by chishaku** [comment]
Looks like an issue with the PIL dependency for recognizing jpegs.

Review these docs to ensure you have all your dependencies:
https://newspaper.readthedocs.io/en/latest/user_guide/install.html#install

On ubuntu, make sure to:
`sudo apt-get install libjpeg-dev zlib1g-dev libpng12-dev`

On osx:
`brew install libtiff libjpeg webp little-cms2`

**Comment by yprez** [comment]
@mansoorfayyaz does installing PIL solve your issue?

**Comment by mansoorfayyaz** [comment]
newspaper3k package already comes with the pillow 4.1.0 for PIL. However, it still gave the same error. 
Referring to stackoverflow, I resolved the issue by first  installing a previous version of pillow 4.0.0 and then installing newspaper3k. This solved the issue for me. Looks like the new pillow 4.1.0 has some dependency issues.
http://stackoverflow.com/questions/43264773/pil-dll-load-failed-specified-procedure-could-not-be-found

I just saw that pillow 4.1.1 is also out, but didnt check it, perhaps it has already addressed this problem.

Thanks guys and I hope this workaround helps.
